server/controllers/users.js

```diff
@@ -32,7 +32,7 @@ export function getUserById(req, res, next) {
     id: req.params.id
   };
 
-  return usersService.getUserById(params)
+  return usersService.getUserById(params, UsersModel)
```

Comment:
Is there a matching method change in `usersService`? Or was this a change to fix a previously breaking change?